### ISSUE TITLE
Inclusion in F-Droid

### ISSUE THREAD

**Issue by Poussinou** [7LPdWcaW/GrowTracker-Android#60]
Hi,

Since this app is Apache v2 licensed, are you considering the fact to add it to [F-Droid](https://f-droid.org/)? There is a strong community behind, that do not use Google Play nor anything else, except F-Droid.

You can find some documentation [here](https://f-droid.org/docs/).

Otherwise, if you don't want to work on the inclusion of this app by yourself, would it be ok for you if someone else do it for you?

**Comment by 7LPdWcaW** [comment]
I've looked at using f-droid before but never managed to get it working properly. I will give it another look this weekend.

**Comment by Poussinou** [comment]
Great ! :)
The documentation is very helpful, if you need it. Otherwise ping me, maybe I could help you, even if I'm not a master in Android development nor anything else

**Comment by Poussinou** [comment]
JAR files like this one [here](https://github.com/7LPdWcaW/GrowTracker-Android/blob/master/app/libs/lombok.jar) are problematic, they should be built from source or taken from a trusted maven repo if they are open-souce.
Otherwise, if they are proprietary, they should be removed.

**Comment by 7LPdWcaW** [comment]
I've gone through the docs and got stuck at running makebuildserver as the 'vagrant' module wasnt found and I havent been able to install it properly.

I've responded to the RFP, but if you wana drop me an email 7lpdwcaw@gmail.com with any details or things that need to be done then im happy to provide details.

Is a shame that they self sign the APK meaning people cant download from here and upgrade via f-droid, but thats fine, as long as its always built from source each time.

I have also imported the project into gitlab here https://gitlab.com/7lpdwcaw/GrowTracker-Android
